### ISSUE TITLE
/notifications call fails with 500 in 1.13.9

### ISSUE THREAD

**Issue by srameshr** [chatwoot/chatwoot#1786]
**Describe the bug**

Visting https://chatwoot.org.xyz/app/accounts/1/notifications fails with the below error:

**To Reproduce**

1. Go to  https://chatwoot.yourord.xyz/app/accounts/1/notifications
2. Click on notifications on the bottom left.
4. See an error in the console

**Expected behavior**

Should not throw an error.

**Screenshots**

NA

**Browser logs**

**NoMethodError in Api::V1::Accounts::Notifications#index
undefined method `display_id' for nil:NilClass**

```

 I18n.t('notifications.notification_title.conversation_creation', display_id: primary_actor.display_id, inbox_name: primary_actor.inbox.name)
    when 'conversation_assignment'
      I18n.t('notifications.notification_title.conversation_assignment', display_id: primary_actor.display_id)
    when 'assigned_conversation_new_message'
      I18n.t(
        'notifications.notification_title.assigned_conversation_new_message',

```

**Server logs**

```
23:29:07 backend.1  | ActionView::Template::Error (undefined method `display_id' for nil:NilClass):
23:29:07 backend.1  |      9:     json.array! @notifications do |notification|
23:29:07 backend.1  |     10:       json.id notification.id
23:29:07 backend.1  |     11:       json.notification_type notification.notification_type
23:29:07 backend.1  |     12:       json.push_message_title notification.push_message_title
23:29:07 backend.1  |     13:       # TODO: front end assumes primary actor to be conversation. should fix in future
23:29:07 backend.1  |     14:       if notification.notification_type == 'conversation_mention'
23:29:07 backend.1  |     15:         json.primary_actor_type 'Conversation'
```
**Environment**

Describe whether you are using Chatwoot Cloud (app.chatwoot.com) or a self hosted installation of Chatwoot. If you are using a self hosted installation of Chatwoot describe the type of deployment (Docker/Linux VM installation/Heroku): Self hosted.

**Additional context**

This occurs in 1.13.0

**Comment by pranavrajs** [comment]
@sojan-official Wasn't this fixed in #1768?

**Comment by srameshr** [comment]
@pranavrajs Also please take a look at #1785

**Comment by sojan-official** [comment]
this should be addressed on `v1.13.1`. Let us know if the issue persists
